swap/src/bob/swap.rs

```diff
@@ -370,7 +370,18 @@ where
                     .await
                     .is_err()
                 {
-                    state4.submit_tx_cancel(bitcoin_wallet.as_ref()).await?;
+                    let result = state4.submit_tx_cancel(bitcoin_wallet.as_ref()).await;
+                    if let Err(error) = result {
+                        let json_rpc_err = error
+                            .downcast_ref::<jsonrpc_client::JsonRpcError>()
+                            .ok_or_else(|| anyhow!("Failed to downcast JsonRpcError"))?;
+
+                        if json_rpc_err.code == -27 {
+                            warn!("Failed to send cancel transaction, assuming that is was already included by the other party...");
```

Comment:
what is error code `-27`? Isn't it ONLY when bitcoind already knows the transaction?
In this case, just do an `info`.
A warn is something that the user should be warned about. The user does not need to be warned here, just informed. It all went well.
```suggestion
                            info!("Cancel Transaction is already published");
```

Reply:
The level was changed to `info` but the message remains the same. Probably better to give a bit more information back to the user at this point. We can change this at a later point if it is confusing.